repose-aggregator/src/docs/asciidoc/release-notes.adoc

```diff
@@ -11,6 +11,7 @@
 * https://repose.atlassian.net/browse/REP-5401[REP-5401] - Added support for environment variable substitution in configuration files.
 * https://repose.atlassian.net/browse/REP-6390[REP-6390] - Internal changes to the <<filters/keystone-v2.adoc#, Keystone v2 Filter>> in anticipation of splitting the authorization portion off into it's own filter.
 * https://repose.atlassian.net/browse/REP-6400[REP-6400] - Added the new <<filters/keystone-v2-authorization.adoc#, Keystone v2 Authorization Filter>> which captures the authorization functionality of the <<filters/keystone-v2.adoc#, Keystone v2 Filter>>.
+* https://repose.atlassian.net/browse/REP-6382[REP-6382] - Lots of little versioned docs updates.
```

Comment:
These look good, but it looks like some stories still need to be tagged. I always use the Wiki `Future Release Candidates` section as a guide:
https://repose.atlassian.net/wiki/spaces/REPOSE/pages/46137362/Version+8x+release+notes